index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav id="navbar">
+        <div class="container">
+          <a href="#" class="page_logo">My Logo</a>
+          <div class="navbar-nav">
```

Comment:
- [ ] Please use the same font size as in the design for the logo if you want to use a responsive size [OPtional method sometimes I use ] you can consider this [website](https://web-development.space/tools/px-to-vw/) you can just put the size of the (font, width, ..) you want in the website depending on the design width, you can see viewport section in the website and then you will get `VW` size which changes the size depending on the screen width this my way it is optional but it works most of the time😉.

| Your code  | Figma |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/211920048-19408a41-9245-4bda-a4cf-d00a7dad2a21.png) | ![image](https://user-images.githubusercontent.com/74744787/211920206-6d110c71-5d59-41fa-b14f-10ba2d93eca4.png) |

Reply:
Hello @MeqdamAlqudah ,
Can you clarify more on this? I'm not the explanation for the responsive size🙂